src/textual/drivers/win32.py

```diff
@@ -217,7 +220,7 @@ def __init__(
         loop: AbstractEventLoop,
         app: App,
         exit_event: threading.Event,
-        process_event: Callable[[Event], None],
+        process_event: Callable[[Message], None],
```

Comment:
Perhaps we should update the name of the parameter to reflect the change in signature, possibly the class as well. i.e. `process_message`, and `MessageMonitor`.